Zuul-MSSC/src/main/java/com/clinics/zuul/security/SecurityConfigurationZUUL.java

```diff
@@ -34,11 +34,16 @@ protected void configure(HttpSecurity http) throws Exception{
 				.antMatchers(HttpMethod.DELETE, "/doctor-mssc/doctors/{uuid}/**").access("@userUuidChecker.checkUserUUID(authentication, #uuid)")
 				.antMatchers(HttpMethod.GET, "/doctor-mssc/**").hasAnyRole(Role.DOCTOR, Role.ASSISTANT, Role.SYSTEM_ADMIN)
 
+				//teoretycznie nie musimy przekazywac UUID w sciezce bo patientUUID idzie w body obiektu wizyty,
+				//czy jestesmy tu w stanie wyciagnac UUID z body requesta i porownac z tokenowym,
+				//jeżeli tak to czy ZUUL musi rozumiec obiekt Visit, ktory idzie w body.
+				//jeżeli tak to będziemy potrzebowali VisitDTO
```

Comment:
Bedzie wygodniej jak będzie DTO ale myślę że tym sposobem za chwile będziemy w Zuul potrzebować więcej takich DTO, nie wiem czy nie lepiej narazie sprawdzać to po uuid skoro i tak potrzebujemy to do blokwoania wchodzenia sobie useróœ na konta, jutro trzeba to przemyśleć